app/builders/digest_email_builder.rb

```diff
@@ -1,4 +1,7 @@
+require 'email_builder_helper'
```

Comment:
I see in the next PR you've moved EmailBuilderHelper so the require isn't needed here.  Would it be possible to have that in this commit too?  Then it reduces the diff between the commits.